reporter/cloudwatch.go

```diff
@@ -178,7 +191,8 @@ func metricsData(cfg *config.Config) []*cloudwatch.MetricDatum {
 				}
 			}
 			for _, p := range cfg.Filter.Percentiles(name) {
-				pname := fmt.Sprintf("%s-perc%.3f", name, p)
+				percentileLabel := strings.Replace(strconv.FormatFloat(p*100.0, 'f', -1, 64), ".", "", 1)
```

Comment:
Hmm if we have to do this in more than one place then could we pull `percentileLabel` out into a helper function?